frontend/document/koptinterface.lua

```diff
@@ -14,6 +14,7 @@ local KOPTContext = require("ffi/koptcontext")
 local Persist = require("persist")
 local TileCacheItem = require("document/tilecacheitem")
 local logger = require("logger")
+local textutil = require("util")
 local util = require("ffi/util")
```

Comment:
`textutil` would be a newcomer for naming this old module :)
Best to just use like in many other modules:
```lua
local util = require("util")
local FFIUtil or BaseUtil = require("ffi/util") -- we use any of these 2 names in other modules
```
The only usage in koptinterface of FFIUtil is util.usleep, some commented out util.gettime, and util.isAndroid - so you're allowed to update these 4 unrelated other lines for consistency sake :)

Reply:
Okay, will do. :smile: